ddp/ddp_pipeline/src/main/java/org/mskcc/cmo/ks/ddp/pipeline/util/DDPUtils.java

```diff
@@ -166,6 +174,43 @@ public static String resolvePatientCurrentAge(DDPCompositeRecord compositeRecord
         return anonymizePatientAge(age.intValue());
     }
 
+    public static String getAgeAtSeqDateLogging(String sampleId, String ageAtSeqDate, Long referenceInDays) {
```

Comment:
Because this is only called when AgeAtSeqDate is null, perhaps the function name should be getAgeAtSeqDateNullLogging() ?
Otherwise it sounds like a general purpose log string generator for any sample